src/components/forms/fields/InputField/InputField.tsx

```diff
@@ -43,7 +46,8 @@ export const InputField = (props: InputFieldProps) => {
     labelProps = {},
     wrapperProps = {},
     tags = [],
-    tagRemoveCallback,
+    onUpdate = null,
+    onUpdateTags = null,
```

Comment:
Should probably be `undefined` instead of `null` (i.e. just remove the `= null` defaults), since the types for these props use `undefined`.